cilium-etcd-cluster/descriptors.go

```diff
@@ -43,6 +43,7 @@ func CiliumEtcdCluster(namespace, version string, size int) *v1beta2.EtcdCluster
 					OperatorSecret: defaults.CiliumEtcdClientTLS,
 				},
 			},
+			ExtendedArgs: "--auto-compaction-mode=revision --auto-compaction-retention=25000",
```

Comment:
Please configure via env variable so we could change it without requiring to release a new version

Reply:
@tgraf Check the new changes, you can already merge it if you want.